pypesto/objective/amici/amici.py

```diff
@@ -402,6 +402,7 @@ def call_unprocessed(
         edatas: Sequence['amici.ExpData'] = None,
         parameter_mapping: 'ParameterMapping' = None,
         amici_reporting: Optional['amici.RDataReporting'] = None,
+        **kwargs,
```

Comment:
This wouldn't be required anymore if we allow for objective-specific kwargs, right?

Reply:
Yes, I would opt to remove/deprecate it. Only kept it for now to not break anyone's code.

Reply:
The one in `AggregatedObjective`, yes. But the one in `AmiciObjective` is only introduced here.

Reply:
Oops. Removed now.